image/image_linux.go

```diff
@@ -481,28 +482,48 @@ func (s *imageSeeder) deriveInfoForLocalSnaps(f seedwriter.SeedAssertionFetcher,
 	return snaps, s.w.InfoDerived()
 }
 
-func (s *imageSeeder) validationSetKeys() ([]snapasserts.ValidationSetKey, error) {
+func (s *imageSeeder) validationSetsAndRevisionForSnap(snapName string) ([]snapasserts.ValidationSetKey, snap.Revision, error) {
```

Comment:
nitpick: sometimes `...vss` refer to validation sets and sometimes to keys. e.g. allVss and snapVss. Function name suggest sets and returns keys.

Reply:
Good point, I've clarified this a bit